cmd/triggers/generate_test.go

```diff
@@ -372,6 +372,7 @@ func Test_TriggerGenerate_MismatchedFlags(t *testing.T) {
 
 	for name, tt := range tests {
 		t.Run(name, func(t *testing.T) {
+			ctx := slackcontext.MockContext(t.Context())
```

Comment:
🔍 This is good.

Reply:
I'm really enjoying the `slackcontext.MockContext()` function. It's been very handy to consistently sprinkle a mocked context into our tests.